app/translations/en.json

```diff
@@ -145,7 +146,7 @@
     "privacys": [
       {
         "value": true,
-        "text": "High Privacy, all the sensitive information is hidden."
+        "text": "High Privacy, all the sensitive information is hidden. You can Tap in any amount as: 'ZEC -.- - - -' or '$ -.- -' to reveal the value."
```

Comment:
and here:
"High Privacy, all the sensitive information is hidden. Tap to reveal hidden values."
what do you think?